src/frontend/src/lib/components/receive/ReceiveAddresses.svelte

```diff
@@ -99,6 +99,6 @@
 	</ReceiveAddressWithLogo>
 </div>
 
-<button class="primary full center text-center" on:click={modalStore.close}
+<button class="secondary full center text-center" on:click={modalStore.close}
```

Comment:
Sounds wrong to have a unique primary action called secondary.

Reply:
yes, it seemed off, but I am not really sure how to call it. That class will be used:
1. By secondary buttons (Close, Cancel, etc) when there is a primary
2. By buttons that are alone in a modal as "definitive" action (Done, Ok, Close, etc)

You think I shall create a new class for the second case?

Reply:
UX wise this seems incorrect no? Same design for "cancel" on certain screen but, used for "confirmation" on other screen? Is that correct @ artkorotkikh-dfinity?